remote/advancedcontainers/docker-options.md

```diff
@@ -0,0 +1,112 @@
+---
+Order: 16
+Area: advancedcontainers
+TOCTitle: Docker Options
+PageTitle: Remote - Containers Docker options
+ContentId:
+MetaDescription: Remote - Containers Docker options
+DateApproved: 2/4/2022
+---
+
+# Alternate ways to install Docker
+
+You may harness Docker in the Remote - Containers extension in a few ways:
+* Docker installed locally
+* Docker installed on another machine or remote environment
+* Other Docker compliant CLIs, installed locally or in a remote environment
+     * Remote - Containers interacts with CLIs; it makes no assumptions about how a container engine works and does not interact with container engines or daemon directly.
+     * Please note that other Docker compliant CLIs are not officially supported.
+
+Continue reading to learn alternate ways you may install and leverage Docker or a Docker compliant CLI.
+
+## Podman
+
+[Podman](https://podman.io/) 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the `Docker Path` setting (which may be accessed through **Remote > Containers: Docker Path**) to `podman` on Linux.
+
+![Docker Path setting](images/platform-options/docker-path-setting.png)
+
+However, certain tricks like [Docker-from-Docker do not work](https://github.com/containers/libpod/issues/4056#issuecomment-535511841) due to limitations in Podman. This affects the **Remote-Containers: Try a Development Container Sample...** and **[Remote- Containers: Clone Repository in Container Volume...](/docs/remote/containers.md#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume)** commands.
+
+Docker Compose is also not supported by Podman.
+
+## Remote hosts
+
+Connecting to a remote host can enable you to connect to a remote environment with Docker or a Docker complaint CLI, rather than needing to install Docker locally.
+
+### Platform options
+
+Docker is available on a variety of Linux platforms, macOS, and Windows. If connecting to a remote Linux machine, you may review how to install Docker [here](https://docs.docker.com/engine/install/), with specific information per distro starting [here](https://docs.docker.com/engine/install/centos/).
+
+### SSH
+You may use the [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) with Remote - Containers. This enables you just to have Docker installed on your remote machine.
+
+You may use Remote - SSH to connect to a Linux VM with Docker installed. As an example, you can create an Azure VM through the Azure CLI, and set it to use a `cloud-init.txt` file to install Docker automatically.
```

Comment:
Maybe mention that Cloud-Init is an industry standard (https://cloud-init.io) and position Azure as "for example" in the following paragraph to indicate that other clouds support this too.

Reply:
I updated the "Cloud-Init VM" section to link to the cloud-init.io, mention it's an industry standard, and keep anything Azure-specific as "for example." Let me know if it looks good to you.

Reply:
LGTM!